### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by Caellian** [brndnmtthws/conky#1800]
It would be nice if someone took some time to go over all settings and improved their documentation.

TBC, many settings have good documentation, but some items are missing `default`s, some `args` are inconsistent, and a few descriptions too short or misleading (e.g. `show_graph_scale`).

This is best done in bulk to avoid many merge conflicts.

**Comment by rubyFeedback** [comment]
I agree. I picked up conky many years ago and over time, I tend to forget, so having a nice reference may be great.
